src/core-class-example.js

```diff
@@ -0,0 +1,27 @@
+class Module {
+  constructor(command) {
+    if (!command || typeof command !== 'string') {
+      throw Error('Command must be a string.');
+    }
+    this.command = command;
```

Comment:
Why do you suggest to add commands storing in modules?
Since we use them only for registration, I don't think that it is necessary.

Reply:
Fixed.